packages.yml

```
@@ -6,4 +6,5 @@ packages:
     install_prerelease: false
     ## Docs: https://docs.elementary-data.com
   - git: https://github.com/tuva-health/dbt-data-reliability.git
-    revision: "v0.18.3"
+    revision: "v0.18.4"
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)